lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -54,6 +54,8 @@ def __init__(self, app, nworkers, **kwargs):
             k8s_job_api_version=dict(map=str, default="batch/v1"),
             k8s_supplemental_group_id=dict(map=str),
             k8s_pull_policy=dict(map=str, default="Default"),
+            k8s_run_as_user_id=dict(map=int, default=os.getuid()),
```

Comment:
@natefoo Is this ok in terms of caching? The assumption is that the job runner is instantiated only once and therefore, this will get cached? The alternative could be to add an app.config.uid property.

Reply:
You're correct about it instantiating once, so this method looks fine to me.